lib/hexpm_web/templates/package/versions.html.eex

```diff
@@ -12,9 +12,11 @@ version_map = Enum.map(@all_releases, & &1.version)
       <%= if release.has_docs do %>
         <a href="<%= raw Hexpm.Utils.docs_html_url(@package.repository, @package, release) %>" title="Docs"><%= icon(:octicon, :book) %></a>
       <% end %>
-      <%= if previous_version = Hexpm.Utils.previous_version(release.version, version_map) do %>
-        <a href="<%= raw Hexpm.Utils.diff_html_url(@package.name, release.version, previous_version) %>" title="Diff"><%= icon(:octicon, :diff) %></a>
-      <% end %>
+      <!-- 
```

Comment:
We should only hide the link if it's a private package. You can check `@package.repository_id == 1`, if it returns true it's a public package.

Reply:
I was wondering just that. Updating my pull request, thanks for the tip!